src/wx/opts.cpp

```diff
@@ -759,9 +762,11 @@ void update_opts()
     if (gopts.accels.size())
         cfg->SetPath(wxT("/Keyboard"));
 
+    int cmd_id = -1;
```

Comment:
Not sure if `-1` is the best value here... I needed to be something like "empty" and that should always be set on the first iteration of the loop. It works for now.